aten/src/ATen/native/cpu/BatchLinearAlgebraKernel.cpp

```diff
@@ -0,0 +1,72 @@
+#include <ATen/ATen.h>
```

Comment:
If you don't expect your code to benefit from AVX instructions, don't put in in the `cpu` folder - it will be compiled multiple times for different avx flavors. In this case, I don't think you can expect benefits - most of what you are doing is calling into precompiled lapack functions.

Reply:
I see, didn't realize that. I'll move the file up one level